it/controllers/agent/business/WhatYearToSignUpControllerISpec.scala

```
@@ -156,10 +148,7 @@ class WhatYearToSignUpControllerISpec extends ComponentSpecBase {
 
           Given("I setup the Wiremock stubs")
           AuthStub.stubAuthSuccess()
-          IncomeTaxSubscriptionConnectorStub.stubSubscriptionData(
-            subscriptionData(incomeSource = Some(IncomeSourceModel(true, false, true))))
           IncomeTaxSubscriptionConnectorStub.stubGetSubscriptionDetails(SelectedTaxYear, OK, Json.toJson(Some(SubscriptionDetailsAccountingYearCurrent)))
-          IncomeTaxSubscriptionConnectorStub.stubSaveSubscriptionDetails[AccountingYear](SubscriptionDataKeys.SelectedTaxYear, userInput)
```

It might be go to replace all `stubSaveSubscriptionDetails` in this file with a new function `stubSaveAccountingYear` similar to this one:
```
  def stubSaveProperty(property: PropertyModel): Unit = {
    when(method = POST, uri = subscriptionUri(Property), body = Json.toJson(property))
      .thenReturn(Status.OK)
  }
```
The tests are successful but the call to save the accounting year is failing silently.